Source/CarthageIntegration/update.bats

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bats
+
+setup() {
+    cd $BATS_TMPDIR
+    rm -rf Ra
+    git clone https://github.com/younata/Ra.git
+    cd Ra
+}
+
+teardown() {
+    cd $BATS_TEST_DIRNAME
+}
+
+@test "carthage update builds everything" {
+    run carthage update --platform mac --no-use-binaries
+    [ "$status" -eq 0 ]
+    [ -e Carthage/Build/Mac/Quick.framework ]
+    [ -e Carthage/Build/Mac/Nimble.framework ]
+}
```

Comment:
Ideally this would build something that has (1) recursive dependencies and (2) both public and private dependencies.